resources/js/router.js

```diff
@@ -22,6 +25,20 @@ export const routes = [
     name: 'home',
     component: Home
   },
+  {
+    path: '/profile',
+    name: 'profile',
+    component: UsersView,
+    props: () => {
+      return {
+        config: {
+          id: store.state.session.currentUser ? store.state.session.currentUser.id : 0,
+          type: 'profile'
+        }
+      };
+    },
+    meta: { requiresAuth: true, passwordConfirmation: true }
```

Comment:
If the user wants to edit the attributes, he could also edit the account via the user manager. The password confirmation isn't required on the other route. There could also be the possibility to bypass this check, as it is just client side.

Reply:
It is not in client side! There is a request send to the backend where the time of last confirmation, which is stored in the session, gets checked. Depending on that the confirmation is rerequested or not.

Reply:
I removed the password confirmation and will create a ticket to implement this correctly.